srtcore/epoll.h

```diff
@@ -111,7 +108,7 @@ friend class CRendezvousQueue;
       /// @param [in] u UDT socket ID.
       /// @return 0 if success, otherwise an error number.
 
-   int remove_usock(const int eid, const SRTSOCKET& u);
+   int remove_usock(const int eid, const SRTSOCKET& u) { static const int Null(0); return update_usock(eid, u, &Null);}
```

Comment:
Couldn't it be simpler if this simply calls:
- EID to CEPollDesc dispatching
- erasing from *Wait
- erasing from *Set

At least it won't do any useless conditional checks.

Reply:
Yes it's exactly what's is done in update_ssock (nothing more),  so we have think good for easy maintenance to factorize add_ssock, update_ssock and remove_ssock in a same function (udpate_ssock) with robust and unify checking:
[epoll.cpp#L244-L245](https://github.com/thomasjammet/srt/blob/epoll-edge/srtcore/epoll.cpp#L244-L245)

Reply:
Just an opinion. It's not a critical function, and this only costs one extra condition check.